compiler+runtime/include/cpp/jank/runtime/obj/repeat.hpp

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include <jank/runtime/behavior/seqable.hpp>
+
+namespace jank::runtime
+{
+  template <>
+  struct static_object<object_type::repeat> : gc
+  {
+    static constexpr native_bool pointer_free{ false };
+    static constexpr native_integer INFINITE{ -1 };
```

Comment:
Let's lowercase this. I know the Clojure source had this, but jank doesn't use full caps for anything.

Aside from that, everything else looks good!

Reply:
updated.